lib/modules/apostrophe-login/index.js

```diff
@@ -296,12 +297,13 @@ module.exports = {
               return callback(null, false);
             }, 1000);
           }
-          return callback(err, user);
+          return self.disableIfInactive(user, callback);
         });
       });
     };
 
     self.verifyTotp = function(user, done) {
+      self.disableIfInactive(user, done);
```

Comment:
Lack of a return statement here means callbacks will be double-invoked. This must be reconsidered so that if we do disable it we do not continue into the other code below it.

Reply:
Ok, here is a new attempt. What do you think @boutell ?